src/configuration/copy.json

```diff
@@ -0,0 +1,14 @@
+{
+  "welcome_message": {
+    "en": "Welcome to\nPathcheck",
```

Comment:
so noticing that this one has the app name hard coded, meaning that if we ever change the app name then there will be multiple places this needs to be updated.  Likely a low risk as the app name likely wont change often, but wanted to point it out.

Reply:
Yeah, I was debating about adding interpolation to these, but then the hook would not be as simple as we have it. Definitely open to extending it. One question I have is how might we coordinate to get this pattern used across.

Reply:
I think its low enough risk of churn to make the simpler implementation of hard coding preferable. Something we can address if its actually turns out to be an issue.